src/host/configuration.h

```diff
@@ -16,11 +16,13 @@ namespace host
     SGX_RELEASE,
     SGX_DEBUG,
     VIRTUAL,
+    SNP,
```

Comment:
AIUI, this has no effect on the node and is only used by the test infra (`args.enclave_type`)? If that's the case, I think it'd be preferable to have the test infra detect that it's running in a SEV-SNP environment and skip tests accordingly (of course, to be clarified in #4069).